.github/workflows/config/gpt2-ci.yaml

```diff
@@ -1,7 +1,7 @@
 port: 8000
 name: gpt2
 route_prefix: /gpt2
-cpus_per_worker: 2
+cpus_per_worker: 1
```

Comment:
Is it OK to use 2 cores, it will be much faster?

Reply:
Sometimes only one core is available, and one core is enough for test.